wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py

```diff
@@ -54,9 +65,14 @@ def get_all_sites(self):
         return s_Arr
 
     def get_site_by_code(self, site_code):
+        """
+        Get wof site from odm2 database by site code
+        :param site_code: Site Code Ex. 'USU-LBR-Mendon'
```

Comment:
Yea. It will sure help trying to understand what's going on. Often people get mixed up when a model is the `WOF` model or the `ODM2` model.